### ISSUE TITLE
Channel Support

### ISSUE THREAD

**Issue by bodysoda69** [hyPnOtICDo0g/rss-chan#3]
Hey is it possible to add channel support too?

**Comment by hyPnOtICDo0g** [comment]
@bodysoda69 Did you try adding the channel id to `CHAT_ID`? I haven't personally tried it, but lemme know if you did.

**Comment by bodysoda69** [comment]
Ofcourse I did. Still can't find out the purpose of CHAT_ID, bot should be sending messages there if added in config?

Also is there a way to add auth users?

**Comment by hyPnOtICDo0g** [comment]
@bodysoda69 Yes, filling `CHAT_ID` in your config sends the message to that specific chat only. I've written about this in the [wiki](https://github.com/hyPnOtICDo0g/rss-chan/wiki/Configuration#environment-variables). Currently, only Bot owner can manage the feeds, but I could add a feature to add the user IDs of the required users in your config so they can manage the feeds as well. Lemme know if that's necessary.

**Comment by bodysoda69** [comment]
> @bodysoda69 Yes, filling `CHAT_ID` in your config sends the message to that specific chat only. I've written about this in the [wiki](https://github.com/hyPnOtICDo0g/rss-chan/wiki/Configuration#environment-variables). 

Yeah I read that and added the variable hoping it'd work. But unfortunately bot sends feeds only in the chat where it's accessed. Could you check if it's working for you?

>Currently, only Bot owner can manage the feeds, but I could add a feature to add the user IDs of the required users in your config so they can manage the feeds as well. Lemme know if that's necessary.

That'd be awesome to have some SUDO_USERS, also something like AUTHORIZED_CHATS would be great too.

**Comment by hyPnOtICDo0g** [comment]
@bodysoda69 I tested using the channel ID and it works like a charm. Telegram API doesn't differentiate between DMs, groups & channels. Everything goes by an ID. My best guess would be, you forgot to prefix the `-100` in your channel ID or the bot permissions in that specific channel are revoked. Let me know if this solves your issue. Your channel id should be in this format:

`-100<channel_id>`

Note that it should be 14 characters long including the minus sign. (exclude the `<` `>`)

I felt implementing `SUDO_USERS` just for an RSS bot doesn't feel right. Instead fork and clone my mirror bot. I've integrated the RSS module into it so that it comes with authorizing specific users built-in.
